### PR TITLE
 #87: fix the global time rounding 

### PR DESCRIPTION
**Pull request by ttldtor**
I think three decimal places will be enough to know about the effect of portals on time

### CODE REVIEW

**Comment by ttldtor** [comment]
Oh. Travis wants at least version 9

File: src/main/scala/ru/org/codingteam/keter/scenes/game/GameViewModel.scala [hunk 1]
```diff
@@ -6,14 +6,14 @@ import ru.org.codingteam.keter.ui.viewmodels.TextViewModel
 class GameViewModel(engine: Engine) {
 
   val map = new GameMapViewModel(engine)
-  val statInfo = new TextViewModel() {
+  val statInfo: TextViewModel = new TextViewModel() {
```
Comment (ForNeVeR) [inline src/main/scala/ru/org/codingteam/keter/scenes/game/GameViewModel.scala]: Why do you need a type annotation here?

File: src/main/scala/ru/org/codingteam/keter/scenes/game/GameViewModel.scala [hunk 2]
```diff
@@ -6,14 +6,14 @@ import ru.org.codingteam.keter.ui.viewmodels.TextViewModel
...
 
     override def text: String = {
       map.universeState.map { universe =>
         universe.player.map { player =>
           s"Faction/Name: ${player.faction.name}/${player.name}\n" +
           s"Health: ${player.stats.health}\n" +
-          s"Time global/local: ${universe.globalEvents.timestamp}/${player.eventQueue.timestamp}"
+          f"Time global/local: ${universe.globalEvents.timestamp}%.03f/${player.eventQueue.timestamp.toLong}"
```
Comment (ForNeVeR) [inline src/main/scala/ru/org/codingteam/keter/scenes/game/GameViewModel.scala]: Why are local and global time formatted differently?

Comment (ForNeVeR) [inline src/main/scala/ru/org/codingteam/keter/scenes/game/GameViewModel.scala]: We've decided that this needs to be consistent.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

**Comment by ForNeVeR** [comment]
Thanks for help!

**Review by ForNeVeR — APPROVED** [review]

**Comment by ttldtor** [comment]
@ForNeVeR merge, please. I have no such rights: `You’re not authorized to merge this pull request. `

**Pull request merged**